src/bot/commands/random/trivia.ts

```diff
@@ -110,20 +121,21 @@ export class Trivia extends Command {
     //         //let c = await parseResponse('https://opentdb.com/api_count.php?category=' + (i + 8).toString() );
     //     }
     // }
-
+    
     async execute(input: Input) {
-
+        
         let category = input.getArgument('category');
         let difficulty = input.getArgument('difficulty');
         let type = input.getArgument('type');
-
+        
         //Translate user input into URLS for the database
         let difficultyURL: string = '&difficulty=' + difficulty;
```

Comment:
```suggestion
        let difficultyURL = '&difficulty=' + difficulty;
```
FYI: You don't need to declare a type here; it is inferred as a string because you are immediately setting it as a string.